app/models.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, Integer, String, Float, ForeignKey, DateTime
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+import datetime
+
+Base = declarative_base()
+
+class City(Base):
+    __tablename__ = 'cities'
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String, unique=True, index=True)

```

Comment:
The `name` field in the `City` model is marked as `unique`, which is good for ensuring no duplicate city names. Ensure that this constraint is handled properly in the application logic to avoid exceptions during city creation.